### PR TITLE
Changes to WordPiece default, updating examples

### PR DESCRIPTION
**Pull request by jessechancy**

### CODE REVIEW

File: examples/nlp/md/text_generation_gpt.md [hunk 1]
```diff
@@ -139,7 +141,10 @@ lower-case and do other irreversible preprocessing operations.
 
 ```python
 tokenizer = keras_nlp.tokenizers.WordPieceTokenizer(
-    vocabulary=vocab, sequence_length=SEQ_LEN
+    vocabulary=vocab,
+    sequence_length=SEQ_LEN,
+    lowercase=True,
+    strip_accents=True,
```
Comment (mattdangerw) [inline examples/nlp/md/text_generation_gpt.md:147]: same here

**Review by mattdangerw — COMMENTED** [review]
one small comment!

File: examples/nlp/md/text_generation_gpt.md [hunk 2]
```diff
@@ -124,6 +124,8 @@ representing the beginning of each line of training data.
 vocab = keras_nlp.tokenizers.compute_word_piece_vocabulary(
     raw_train_ds,
     vocabulary_size=VOCAB_SIZE,
+    lowercase=True,
+    strip_accents=True,
```
[the remark this answers is not in the document]
Comment (jessechancy) [reply examples/nlp/md/text_generation_gpt.md]: edited

File: examples/nlp/md/text_generation_gpt.md (hunk 1 above)
Comment (jessechancy) [reply examples/nlp/md/text_generation_gpt.md]: edited

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**